pnpm-lock.yaml

```diff
@@ -233,7 +233,7 @@ importers:
       broccoli-asset-rev: ^3.0.0
       ember-addon-automated-ci: 2.1.4
       ember-aria-utilities: '*'
-      ember-auto-import: ^2.4.3
+      ember-auto-import: ^2.4.1
```

Comment:
this should resolve to the same version 🙃

Reply:
I got an error about this not resolving to 2.4.3, hence why I tried downgrading. But this doesn't seem to fix it so I will roll back.

As an aside, I `rm -rf node_modules` from all the packages, then `rm pnpm-lock.yaml && pnpm install`, then `pnpm --filter test-app test:ember` and tests all seemed to pass?

Reply:
oh! but wait....this is now giving me another `pnpm-lock.yaml` file...

Reply:
> pnpm --filter test-app test:ember

this doesn't use the embroider scenario, the other scripts in the test-app's package.json would handle that.

Reply:
ah okay, I was hoping it would shed light on why the Floating Deps test failed https://github.com/CrowdStrike/ember-aria/blob/main/.github/workflows/ci.yml#L136